boutdata/squashoutput.py

```diff
@@ -238,7 +237,6 @@ def squashoutput(
                 f.write(varname, var)
 
         var = None
-        gc.collect()
```

Comment:
I suspect these `gc.collect()` calls are here because someone was running out of memory when squashing data output. OK to remove, or keep as an option?

Reply:
I think OK to remove. Machines generally have more RAM now than when this was a problem (if it was a problem)!